tests/tests/test_qtpy/test_collapsible_widget.py

```diff
@@ -86,61 +102,115 @@ def test_collapsible_widget_container(qtbot, collapsible_widget_container):
     assert len(collapsible_widget_container.collapsible_widgets) == 0
 
 
+@pytest.mark.parametrize(
+    "layout", [QVBoxLayout(), QHBoxLayout(), QFormLayout()]
+)
 def test_collapsible_widget_container_add_collapsible_widget(
-    qtbot, collapsible_widget_container, collapsible_widget
+    qtbot, collapsible_widget_container, generic_widget, layout
 ):
     qtbot.addWidget(collapsible_widget_container)
 
-    collapsible_widget_container.add_widget(collapsible_widget)
+    generic_widget.setLayout(layout)
+    generic_widget.layout().addWidget(QLabel("test"))
+    generic_widget.layout().addWidget(QPushButton("test"))
+
+    collapsible_widget_container.add_widget(
+        generic_widget, widget_title=WIDGET_TITLE
```

Comment:
```suggestion
        generic_widget, collapsible=True, widget_title=WIDGET_TITLE
```
or add a comment stating that `collapsible=True` is the default?
(I know it's the default but makes the test easier to understand, especially given that the name is `generic_widget`?)

Reply:
Always better to be explicit! I've adjusted it where necessary in the tests.